nemo_skills/prompt/config/robustness/prompt_set_config.yaml

```diff
@@ -1,46 +1,25 @@
 gpqa:
```

Comment:
might be better to use a dict here with explicit names like prompt: extract_regex: . This way you can have a dataclass inside starting script and validate names and will also make it easier to extend with more parameters in the future